app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/interview/InterviewViewModel.kt

```diff
@@ -84,6 +85,13 @@ class InterviewViewModel @Inject constructor(
         }
     }
 
+    fun upDateInterviewStatus(status: InterviewStatus){
+        interviewData = interviewData.copy(
```

Comment:
just a minor thing, maybe we could change the function name to updateInterviewStatus